exercises/concept/currency-exchange/.docs/instructions.md

```diff
@@ -80,22 +74,17 @@ This function should return the maximum value you can get considering the `budge
 95
```

Comment:
```suggestion
80
```

Reply:
If we take the exemplar code :

```python
def exchangeable_value(budget, exchange_rate, spread, denomination):
    """

    :param budget: float - the amount of your money you are planning to exchange.
    :param exchange_rate: float - the unit value of the foreign currency.
    :param spread: int - percentage that is taken as an exchange fee.
    :param denomination: int - the value of a single bill.
    :return: int - maximum value you can get.
    """

    exchange_fee = (exchange_rate / 100) * spread
    actual_rate = exchange_rate + exchange_fee
    exchangeable_amount = int((budget / actual_rate) / denomination)
    return exchangeable_amount * denomination
```

and plug in the example `exchangeable_value(127.25, 1.20, 10, 20)` , the result is `80`, not `95`.

Reply:
```python
exchangeable_value(127.25, 1.20, 10, 5)
95
```
```python
exchangeable_value(127.25, 1.20, 10, 20)
80
```

I ran the tests myself as well, these two are correct on my machine. It is true that `exchangeable_value(127.25, 1.20, 10, 20)` is not `95`, but I think you might've grabbed the wrong call, because in the instructions it says `exchangeable_value(127.25, 1.20, 10, 5)` == `95`. Or am I seeing this wrong myself?

Reply:
No -- you are correct, @J08K  -- the spacing in the review window fooled me, and I should have viewed the whole file.  The `80` is indeed in the correct place.  😊